### PR TITLE
Constructs and verifies signed_data

### PR DESCRIPTION
**Pull request by bjornvolcker**

### CODE REVIEW

File: lib/vendors/axis-communications/sv_vendor_axis_communications.c [hunk 1]
```diff
@@ -75,6 +75,8 @@ static const uint8_t kAttributes[ATTRIBUTES_LENGTH] = {0x7b, 0x00, 0x02, 0x01, 0
 static const uint8_t kFreshness[FRESHNESS_LENGTH] = {
     0x92, 0xbb, 0xed, 0xfb, 0x98, 0x82, 0xac, 0x16, 0xc7, 0xf0, 0x1a, 0xe4, 0x59, 0x05, 0x96, 0x04};
 
+#define SIGNED_DATA_SIZE (HASH_DIGEST_SIZE + 52 + PUBLIC_KEY_UNCOMPRESSED_SIZE + ATTRIBUTES_LENGTH)
```
Comment (lusikamalo) [inline lib/vendors/axis-communications/sv_vendor_axis_communications.c:78]: Patch looks good but I can't resist to ask where '52' comes from.

Comment (bjornvolcker) [reply lib/vendors/axis-communications/sv_vendor_axis_communications.c:78]: Taken out of the blue...
I'll refine it, and add a comment.

**Review by Caroline-Holmstrom — APPROVED** [review]
LGTM

**Review by lusikamalo — APPROVED** [review]
OK.

**Pull request merged**